### PR TITLE
Xata Go SDK annoucment

### PR DESCRIPTION
**Pull request by philkra**
## Summary

- Blog post announcing the xata-go SDK

### Blog Release Checklist

- \[x] Release on: 12/06/2023
- \[ ] Published on dev.to (connected to Xata org)
- \[x] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: announcing-xata-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  alt: Xata Go SDK
+author: Philip Krauss
+date: 12-06-2023
+published: true
+slug: announcing-xata-go-sdk
+---
+
+We thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community to Xata with an official SDK.
+The `v0.0.1` release will feature the most popular endpoints out of the box, to allow you to search a branch, use the ask endpoint with follow up questions, create a database, you name it.
+You can see the evolving endpoint coverage in the ticket [xata-go#1](https://github.com/xataio/xata-go/issues/1).
```
Comment (alexfrancoeur) [inline announcing-xata-sdk.mdx:15]:
```suggestion
Earlier this year we shared a [community spotlight](https://xata.io/blog/community-spotlight-xata-go-sdk) focused on a community contributed Go SDK, [xata-go](https://github.com/kerdokurs/xata-go). [Kerdo Kurs](https://www.linkedin.com/in/kerdokurs/), the developer behind this project, was a huge inspiration for us committing to an official Go SDK. Today, we're thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community 🎉
The `v0.0.1` release will feature the most popular endpoints out of the box, to allow you to search a branch, use the ask endpoint with follow up questions, create a database, you name it.
You can see the evolving endpoint coverage in the ticket [xata-go#1](https://github.com/xataio/xata-go/issues/1).
```

A few tweaks to try and bring in a Kerdo hat tip, what do you think? @philkra @tsg @joan-ing

File: announcing-xata-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+)
+```
+
+## What’s next?
+
+The SDK is an alpha release, and not all [endpoints are covered](https://github.com/xataio/xata-go/issues/1), we continue to increase the API coverage of the SDK and provide bugfixes.
+
+If you want to contribute [code](https://github.com/xataio/xata-go), add an [example](https://github.com/xataio/xata-go/tree/main/examples) or you found a missing piece the [docs](/docs/sdk/go/overview), please open a PR or ticket and we will assist you with your open-source contribution.
+
+We can't do it without you! If you think something is missing or you found a bug, open a ticket in the [xata-go](https://github.com/xataio/xata-go) repository. All contributions are welcome. You can also follow us on [Twitter](https://twitter.com/xata) or join us in [Discord](https://xata.io/discord).
```
Comment (alexfrancoeur) [inline announcing-xata-sdk.mdx:54]:
```suggestion
We can't do it without you! If you want to contribute [code](https://github.com/xataio/xata-go), add an [example](https://github.com/xataio/xata-go/tree/main/examples) update the [docs](/docs/sdk/go/overview), or found a bug  please open a PR or issue and we will assist you. All contributions are welcome ✅ 

To stay up to date with that latest at Xata, follow us on [X /Twitter](https://twitter.com/xata) or pop in and say hi in [Discord](https://xata.io/discord).
```

Another suggestion, the two paragraphs felt similar so I attempted to combine them

File: announcing-xata-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- `FilesClient`
+- `RecordsClient`
+- `SearchAndFilterClient`
+- `TableClient`
+- `UsersClient`
+- `WorkspaceClient`
+
+The code and corresponding test suites can be found in the [xata package](https://github.com/xataio/xata-go/tree/main/xata).
+
+```go
```
Comment (alexfrancoeur) [inline announcing-xata-sdk.mdx:39]: Are there any other code examples we can provide? Maybe even some from the docs? Or a quick example from each client?

File: announcing-xata-sdk.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+  alt: Xata Go SDK
+author: Philip Krauss
+date: 12-06-2023
+published: true
+slug: announcing-xata-go-sdk
+---
+
+We thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community to Xata with an official SDK.
+The `v0.0.1` release will feature the most popular endpoints out of the box, to allow you to search a branch, use the ask endpoint with follow up questions, create a database, you name it.
```
[the remark this answers is not in the document]
Comment (philkra) [reply announcing-xata-sdk.mdx]: Change to this wording in the editor, thanks @joan-ing

File: announcing-xata-sdk.mdx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+  alt: Xata Go SDK
+author: Philip Krauss
+date: 12-06-2023
+published: true
+slug: announcing-xata-go-sdk
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+---
+
+Earlier this year we shared a [community spotlight](https://xata.io/blog/community-spotlight-xata-go-sdk) focused on a community contributed Go SDK, [xata-go](https://github.com/kerdokurs/xata-go). [Kerdo Kurs](https://www.linkedin.com/in/kerdokurs/), the developer behind this project, was a huge inspiration for us committing to an official Go SDK. Today, we're thrilled to announce the first release of our Xata Go SDK, and to open the doors to the growing Go community 🎉
```
Comment (alexfrancoeur) [inline announcing-xata-sdk.mdx:14]: I don't know how to get away from using the word community twice in this sentence haha. Any ideas @joan-ing ?

Comment (philkra) [reply announcing-xata-sdk.mdx]: addressed via https://github.com/xataio/mdx-blog/pull/165/commits/cbd9730c2d849fa0ddf183310a42ec55909eb8ec

File: announcing-xata-sdk.mdx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+  alt: Xata Go SDK
+author: Philip Krauss
+date: 12-06-2023
+published: true
+slug: announcing-xata-go-sdk
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/announcing-xata-sdk-cover.jpg
+---
+
+Earlier this year we shared a [community spotlight](https://xata.io/blog/community-spotlight-xata-go-sdk) focused on a Go SDK developed by our dedicated (zealous, enthusiastic) contributors, [xata-go](https://github.com/kerdokurs/xata-go). [Kerdo Kurs](https://www.linkedin.com/in/kerdokurs/), the developer behind this project, was a huge inspiration for us committing to an official Go SDK. Today, we're thrilled to announce the official release of our Xata Go SDK, and open the doors to the growing Go community 🎉
```
Comment (joaedwar) [inline announcing-xata-sdk.mdx:14]:
```suggestion
Earlier this year we shared a [community spotlight](https://xata.io/blog/community-spotlight-xata-go-sdk) focused on a Go SDK developed by dedicated contributors, [xata-go](https://github.com/kerdokurs/xata-go). [Kerdo Kurs](https://www.linkedin.com/in/kerdokurs/), the developer behind this project, was a huge inspiration for us committing to an official Go SDK. Today, we're thrilled to announce the official release of our Xata Go SDK, and open the doors to the growing Go community 🎉
```

**Pull request merged**